nucypher/characters/chaotic.py

```diff
@@ -133,15 +134,16 @@ class Felix(Character, NucypherTokenActor):
     TEMPLATE_NAME = 'felix.html'
 
     # Intervals
-    DISTRIBUTION_INTERVAL = 60*60   # seconds (60*60=1Hr)
-    DISBURSEMENT_INTERVAL = 24      # (24) hours
-    STAGING_DELAY = 10              # seconds
+    DISTRIBUTION_INTERVAL = 60  # seconds
+    DISBURSEMENT_INTERVAL = 24 * 365  # only distribute tokens to the same address once each YEAR.
```

Comment:
Hm.. I think, 1 year (!) per disbursement pretty much makes it one-time disbursement in practice though?